extensions/wikidata/src/org/openrefine/wikidata/qa/scrutinizers/MultiValueScrutinizer.java

```diff
@@ -0,0 +1,50 @@
+package org.openrefine.wikidata.qa.scrutinizers;
+
+import org.openrefine.wikidata.qa.QAWarning;
+import org.openrefine.wikidata.updates.ItemUpdate;
+import org.wikidata.wdtk.datamodel.interfaces.PropertyIdValue;
+import org.wikidata.wdtk.datamodel.interfaces.Statement;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class MultiValueScrutinizer extends EditScrutinizer {
+
+    public static final String new_type = "multi-valued-property-is-required-for-new-item";
+    public static final String existing_type = "multi-valued-property-is-required-for-existing-item";
+
+    @Override
+    public void scrutinize(ItemUpdate update) {
+        Map<PropertyIdValue, Integer> propertyCount = new HashMap<>();
+
+        for (Statement statement : update.getAddedStatements()) {
+            PropertyIdValue pid = statement.getClaim().getMainSnak().getPropertyId();
+            if (propertyCount.containsKey(pid)) {
+                propertyCount.put(pid, propertyCount.get(pid) + 1);
+            } else if (_fetcher.hasMultiValue(pid)) {
+                propertyCount.put(pid, 1);
+            }
+        }
+
+        if (update.isNew()) {
+            for (PropertyIdValue pid : propertyCount.keySet()) {
+                if (propertyCount.get(pid) == 1) {
+                    QAWarning issue = new QAWarning(new_type, pid.getId(), QAWarning.Severity.WARNING, 1);
+                    issue.setProperty("property_entity", pid);
+                    issue.setProperty("example_entity", update.getItemId());
+                    addIssue(issue);
+                }
+            }
+        } else {
+            for (PropertyIdValue pid : propertyCount.keySet()) {
+                if (propertyCount.get(pid) == 1) {
+                    QAWarning issue = new QAWarning(existing_type, pid.getId(), QAWarning.Severity.WARNING, 1);
```

Comment:
Should we use a lower severity level for warnings on existing items?

Reply:
Very sorry, I didn't get your point. Can you please explain a bit more?

Reply:
I just thought that because the warnings we raise on existing items can potentially be false positives, we could set them to a lower severity level than `QAWarning.Severity.WARNING`, so that they do not look too scary to users. (The severity level determines the icon used in the issues tab). But that is only a suggestion, it is not a big deal if we keep the same level.